src/test/kotlin/com/zepben/evolve/services/network/NetworkServiceComparatorTest.kt

```diff
@@ -218,8 +215,10 @@ internal class NetworkServiceComparatorTest : BaseServiceComparatorTest() {
         comparatorValidator.validateProperty(
             Location::mainAddress,
             { Location(it) },
-            { StreetAddress("1234", TownDetail("town", "state")) },
-            { StreetAddress("1234", TownDetail("other", "state")) })
+            { StreetAddress("1234", TownDetail("town", "state"), "5678",
```

Comment:
Please put args on seperate lines when wrapping in the JVM SDK

Reply:
If I specify default values for all fields of `StreetAddress`, is this better:
```
comparatorValidator.validateProperty(
            Location::mainAddress,
            { Location(it) },
            { StreetAddress(townDetail = TownDetail("town", "state")) },
            { StreetAddress(townDetail = TownDetail("other", "state")) })
```

Reply:
New revision uses above code.